xxhash.h

```diff
@@ -3865,8 +3865,8 @@ XXH_FORCE_INLINE XXH_TARGET_SSE2 void XXH3_initCustomSecret_sse2(void* XXH_RESTR
 #       endif
         int i;
 
-        XXH_ALIGN(64)        const float* const src  = (float const*) XXH3_kSecret;
-        XXH_ALIGN(XXH_SEC_ALIGN) __m128i*       dest = (__m128i*) customSecret;
+        const __m128i* const src16 = (const __m128i *)XXH3_kSecret;
```

Comment:
As expected, this variant fails the `-Wcast-align` test on older `clang` version, 
as these compilers fail to notice that `XXH3_kSecret` is correctly aligned, 
since they only take into consideration the source type `const unsigned char*`, but not its alignment `XXH_ALIGN(64)`.